webdriver/tests/bidi/browser/set_download_behavior/conftest.py

```
@@ -72,15 +79,28 @@ async def trigger_download(context):
             user_activation=True,
         )
 
-        return await wait_for_future_safe(on_download_will_begin)
+        try:
+            return await wait_for_future_safe(
+                on_download_will_begin, timeout=0.5)
+        except TimeoutException:
+            # User Agent showed file save dialog.
+            return None
```

@sadym-chromium Have you seen this issue yet? Or does Chrome in some cases open the save dialog? If that's the case shall we ensure that WebDriver should force this dialog to not come up at all? It's not great that we silence the error here which makes it hard to investigate why some problem might appear at a later time.

For now we are going to at least print a line when this timeout happens. The change is upcoming via https://bugzilla.mozilla.org/show_bug.cgi?id=1999473.

What do you think? Shall we file a BiDi issue for that?